src/xo-app/vm/tab-advanced.js

```diff
@@ -175,8 +236,35 @@ class CoresPerSocket extends Component {
   }
 }
 
-export default ({
+export default connectStore(() => {
+  const getVgpus = createGetObjectsOfType('VGPU').pick(
+    (_, { vm }) => vm.$VGPUs
+  ).sort()
```

Comment:
Configure sort criterias for all these new types.

https://github.com/vatesfr/xo-web/blob/6a071942a5fda28eedf89256c17a7ac4393b1983/src/common/selectors.js#L330-L339